lib/SPIRV/SPIRVToOCL.cpp

```diff
@@ -582,6 +582,7 @@ void SPIRVToOCLBase::visitCallSPIRVImageMediaBlockBuiltin(CallInst *CI, Op OC) {
 }
 void SPIRVToOCLBase::visitCallBuildNDRangeBuiltIn(CallInst *CI, Op OC,
                                                   StringRef DemangledName) {
+  assert(CI->getCalledFunction() && "Unexpected indirect call");
```

Comment:
As I see in almost all cases Attrs is requested here and passed into mutateCallInstOCL function.
May we add mutateCallInstOCL version without Attrs parameter and initialize this parameter there with the assert check to reduce code duplication?

Reply:
`visitCallBuildNDRangeBuiltIn` is called from `visitCallInst` which early exits in case the called function is `nullptr`.